CMakeLists.txt

```diff
@@ -264,19 +264,16 @@ if (VAST_ENABLE_STATIC_EXECUTABLE AND BUILD_SHARED_LIBS)
 endif ()
 
 if (VAST_ENABLE_RELOCATABLE_INSTALLATIONS AND NOT VAST_ENABLE_STATIC_EXECUTABLE)
+  # Note that we cannot use the target property INSTALL_RPATH on
+  # libvast_internal, because it is an interface library. We should probably fix
+  # this by making the config header part of libvast_internal so it is not just
```

Comment:
Neat idea, but header only libs are still INTERFACE, PUBLIC requires at least one TU.

Reply:
We can add a stub TU then. :)